packages/api/src/routes/medical/internal-facility.ts

```diff
@@ -190,4 +160,67 @@ function buildCqOboOrgName(vendorName: string, orgName: string, oboOid: string)
   return `${vendorName} - ${orgName} #OBO# ${oboOid}`;
 }
 
+async function createOrUpdateInCq(
+  facilityInput: FacilityOboDetails,
+  facility: FacilityModel,
+  cxOrg: OrganizationModel,
+  vendorName: string
+) {
+  const cqOboData = await getCqOboData(facilityInput.cqActive, facilityInput.cqOboOid);
```

Comment:
We created this and had it before the `createFacility` specifically b/c we want to avoid creating the record on our DB if the request is invalid (OID doesn't exist on CQ or org has no name on CQ).

Reply:
💯 good catch! I'll add a comment so we don't forget again